### DISCUSSION TITLE
Possible to enter multiple Approvers on Object?

### DISCUSSION THREAD

**Discussion by ferneaux** [Venafi/venafips#105] [Q&A] [answered]
Hi folks,

Does anyone know of a way to add multiple approvers when creating a new TppObject or even an existing one?

For example... and I know I've leaving a lot out of what is used to create this object, but just want to highlight the point of my question... if I get the values from an existing object I made in the UI and insert them in the command as such (with correct guids though):
`New-TppObject -Path 'path_here\object_name_here' -Class 'F5 LTM Advanced' -Attribute @{
    'Approver'='local:{aaaabbbbccccddddeeeeffff},AD+RJF_ActiveDirectory:111122223333444455556666'
}
`

It will create the F5 LTM object, but only include whichever account is listed first in the 'Approver' attribute. I know that all the guids I try are correct as it will create them properly if I only include a single item.

Should I be formatting this differently within the New-TppObject cmdlet? I'm not too familiar with actual API commands, but could it be done if I leave out the "Approver" attribute from the New-TppObject and follow that command up with using "Invoke-VenafiRestMethod" to add the multiple accounts afterwards?

Any thoughts/ideas?

Thanks!

**Comment by gdbarron** [comment, marked as answer]
Hi @ferneaux.  Just a small typo is the issue.  You are passing in a string instead of an array.  Try `New-TppObject -Path 'path_here\object_name_here' -Class 'F5 LTM Advanced' -Attribute @{ 'Approver'=@('local:{aaaabbbbccccddddeeeeffff}','AD+RJF_ActiveDirectory:111122223333444455556666') }`

**Reply by ferneaux** [reply]
ugh... arrays!!! <_shakes fist in air_>

That did it... thank you, as always!
